src/main.ts

```diff
@@ -101,12 +102,13 @@ export function createEmitter<T extends Events>({ broadcastChannel: useBroadcast
     }
 
     const event = globalHandlerOrEvent
+    const eventHandlers = handlers.get(event)
 
-    if (!handler) {
-      throw new Error(`Handler must be given for ${String(event)} event`)
+    if (handler) {
+      eventHandlers?.delete(handler)
+    } else {
+      eventHandlers?.clear()
```

Comment:
```suggestion
    if (handler) {
      eventHandlers?.delete(handler)
      return
    }
    
    eventHandlers?.clear()
```

Reply:
Haha I knew you would make this suggestion!

Reply:
death to `else` haha